packages/sequelize-transparent-cache/lib/cache/util.js

```diff
@@ -24,6 +29,21 @@ function dataToInstance (model, data) {
   return instance
 }
 
+function loadAssociations (model) {
+  let associations = []
+  Object.keys(model.associations).forEach((key) => {
+    const association = {}
+    //  model.associations[key] does not work on include, we grab it from sequelize.model()
+    if (model.associations[key].hasOwnProperty('options')) {
+      let modelName = model.associations[key].options.name.singular
+      association.model = model.sequelize.model(modelName)
+      association.as = key
+      associations.push(association)
```

Comment:
how about 

```javascript
associations.push({  
  model: model.sequelize.model(modelName),
  as: key
})
```

Reply:
I'll make this change. Thanks for the feedback!